core/src/main/java/org/apache/iceberg/util/LockManagers.java

```diff
@@ -37,7 +37,7 @@
 
 public class LockManagers {
 
-  private static final LockManager LOCK_MANAGER_DEFAULT = new InMemoryLockManager(Maps.newHashMap());
```

Comment:
there is already a `defaultLockManager` method below exposing this, we don't need to make it public